tgui/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "private": true,
   "name": "tgui-workspace",
-  "version": "4.3.1",
+  "version": "4.3.2",
```

Comment:
this should probably be a minor version bump not a build version bump

Reply:
sry these were from #79895

the ~only~ change here is the one in webpack.config.js

Reply:
(but I agree)